docs/configuration.md

```diff
@@ -80,6 +115,41 @@ sentinels.
 - Applies to: listener, worker, cron.
 - Format: string.
 
+###  CONFIG_QUEUES_USERNAME
+
+- Redis ACL user name
+- Optional. Defaults to empty.
+- Applies to: listener, worker, cron.
+- Format: string.
+
+###  CONFIG_QUEUES_PASSWORD
+
+- Redis ACL password
+- Optional. Defaults to empty.
+- Applies to: listener, worker, cron.
+- Format: string.
+
+###  CONFIG_QUEUES_CA_FILE
+
+- Certification authority certificate Redis should trust to accept TLS connections
+- Optional. Defaults to empty.
+- Applies to: listener, worker, cron.
+- Format: path to file as string.
+
+###  CONFIG_QUEUES_CERT
+
+- User certificate to connect to Redis through TLS
+- Optional. Defaults to empty.
+- Applies to: listener, worker, cron.
+- Format: path to file as string.
+
+###  CONFIG_QUEUES_KEY
```

Comment:
I wonder if all of these make more sense to have standard file locations. And users would be able to mount secrets at these locations with the file contents. Because these files will still need some way to be mounted into the containers anyway. So it would be double the work. Not sure, just a thought.

Reply:
I think it's good to have the variables, since they can be very useful for development: the standard file locations you mention would probably be out of the project folder. What we could do is adding a default in the case they are not provided.

Reply:
This is what I meant.

Reply:
Done in https://github.com/3scale/apisonator/pull/350/commits/a81a4e9d2b45b49c6d8be418963de9dc2c03664e

Reply:
will we have these variables ever empty? Just wondering how it will work. As if empty, I assume empty value may confuse us. Or not?

Reply:
I pushed some changes to ensure empty variables doesn't reach Redis: https://github.com/3scale/apisonator/pull/350/commits/c429e716213fe2ff7c9a66d0d838188f51c13ec4.